modules/logging-content-filter-prune-records.adoc

```diff
@@ -1,54 +1,80 @@
 // Module included in the following assemblies:
 //
 // * observability/logging/performance_reliability/logging-content-filtering.adoc
+// * configuring/configuring-log-forwarding.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="logging-content-filter-prune-records_{context}"]
 = Configuring content filters to prune log records
 
-When the `prune` filter is configured, the log collector evaluates log streams according to the filters before forwarding. The collector prunes log records by removing low value fields such as pod annotations.
+If you configure the `prune` filter, the log collector evaluates log streams against the filters before forwarding. The collector prunes log records by removing low value fields such as pod annotations.
 
 .Prerequisites
 
 * You have installed the {clo}.
 * You have administrator permissions.
 * You have created a `ClusterLogForwarder` custom resource (CR).
+* You have installed the {oc-first}.
 
 .Procedure
 
-. Add a configuration for a filter to the `prune` spec in the `ClusterLogForwarder` CR.
+. Extract the existing `ClusterLogForwarder` configuration and save it as a local file.
 +
-The following example shows how to configure the `ClusterLogForwarder` CR to prune log records based on field paths:
+[source,terminal]
+----
+$ oc get clusterlogforwarder <name> -n <namespace> -o yaml > <filename>.yaml
+----
++
+Where:
++
+* `<name>` is the name of the `ClusterLogForwarder` instance you want to configure.
+* `<namespace>` is the namespace where you created the `ClusterLogForwarder` instance, for example `openshift-logging`.
+* `<filename>` is the name of the local file where you save the configuration.
```

Comment:
```suggestion
* Replace `<name>` with the name of the `ClusterLogForwarder` instance you want to configure.
* Replace `<namespace>` with the namespace where you created the `ClusterLogForwarder` instance, for example `openshift-logging`.
* Replace `<filename>` with the name of the local file where you save the configuration.
```

- https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text
> Do not start a sentence with a lowercase term; rewrite the sentence.

- Up to you --  You can use call outs and remove `where`